exec/java-exec/src/test/java/org/apache/drill/exec/store/parquet/TestParquetBulkReader.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.exec.store.parquet;
+
+import org.apache.drill.test.BaseTestQuery;
+import org.apache.drill.exec.ExecConstants;
+import org.apache.drill.exec.planner.physical.PlannerSettings;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/** Tests the Parquet bulk reader */
+@SuppressWarnings("deprecation")
```

Comment:
One minor comment: since `BaseTestQuery` is deprecated, I suppose we should try to write unit tests using `ClusterTest` rather then suppression the warning.

Reply:
@arina-ielchiieva,
- Majority of the tests are using BaseTestQuery; I even found existing tests easier to write than with ClusterTest (setting session flags, no need to define upfront schema, result-set validation). My guess, we should create a JIRA to move all existing BaseTestQuery to ClusterTest so that DEV get a chance to get more familiar with this test harness.
- I use Eclipse; the UI shows warnings all over the place
- Suppressing the deprecation allows me to focus on more important warnings

Reply:
@arina-ielchiieva 
Thank you for the info and the sample test! that really helped.. I have now ported the new Parquet test to using the ClusterTest base class.